package.json

```diff
@@ -11,52 +11,40 @@
     "cli"
   ],
   "homepage": "https://github.com/interactivethings/gsheets",
+  "author": "Jeremy Stucki - Interactive Things, Olivier Baumann - Republik Magazin",
```

Comment:
That's not according to spec ;-) Add `contributors` instead and leave `author` as is.

> The "author" is one person. "contributors" is an array of people. A "person" is an object with a "name" field and optionally "url" and "email", like this:
https://docs.npmjs.com/cli/v7/configuring-npm/package-json#people-fields-author-contributors